### PR TITLE
On mobile, improve UI to show the minimum budget can be lowered

### PR DESCRIPTION
**Pull request by Ovifer13**
**Issue #252**

- [x] Show that minimum budget can be lowered on mobile

**Proof**
image

### CODE REVIEW

File: components/BudgetTimelineForm.tsx [hunk 1]
```diff
@@ -165,10 +167,10 @@ const BudgetTimelineForm = ({
...
                         }}
                         handleStyle={{
-                            height: 28,
-                            width: 28,
-                            marginLeft: -14,
-                            marginTop: -14,
+                            height: size.screen == 'xs' ? 18 : 28,
+                            width: size.screen == 'xs' ? 18 : 28,
+                            marginLeft: size.screen == 'xs' ? -5 : -14,
+                            marginTop: size.screen == 'xs' ? -8 : -14,
```
Comment (otech47) [inline components/BudgetTimelineForm.tsx:173]: are these breakpoints the same that we use in our styling system?

**Review by otech47 — CHANGES_REQUESTED** [review]
definitely a bit better, can we maybe make the color of the background line a darker grey? since it's so light grey it's still a bit hard to see at least on my screen... like literally just a few shades darker grey should do the trick @Ovifer13

File: components/BudgetTimelineForm.tsx (hunk 1 above)
Comment (Ovifer13) [reply components/BudgetTimelineForm.tsx:173]: No, I checked the styling files and didn't see any breakpoints there, I had to play with the sizes to see which one was the best match.

**Comment by Ovifer13** [comment]
@otech47 Do that grey work?
image

File: components/BudgetTimelineForm.tsx [hunk 2]
```diff
@@ -162,13 +171,14 @@ const BudgetTimelineForm = ({
                         onChange={onSliderChange}
                         step={500}
                         railStyle={{
-                            height: 2
+                            height: 2,
+                            backgroundColor: '#C5C5C5'
```
Comment (otech47) [inline components/BudgetTimelineForm.tsx:175]: really should avoid hard-coding colors in general but fine for now i guess

**Review by otech47 — APPROVED** [review]

**Pull request merged**